### PR TITLE
Xcode 12 build

### PR DESCRIPTION
**Pull request by rickpasetto**
These are just minor changes based on how Xcode 12 saves scheme files

### CODE REVIEW

File: Loop.xcodeproj/xcshareddata/xcschemes/Complication - WatchApp.xcscheme [hunk 1]
```diff
@@ -90,8 +90,8 @@
       launchAutomaticallySubstyle = "32">
       <RemoteRunnable
          runnableDebuggingMode = "2"
-         BundleIdentifier = "com.apple.carousel"
-         RemotePath = "/Naterade">
+         BundleIdentifier = "com.apple.Carousel"
+         RemotePath = "/$(MAIN_APP_DISPLAY_NAME)">
```
Comment (darinkrauss) [inline Loop.xcodeproj/xcshareddata/xcschemes/Complication - WatchApp.xcscheme]: Have you actually gotten the debugger to launch with the watch using this? I've been having problems lately with 11.7.

**Review by darinkrauss — APPROVED** [review]
On question, but assuming you've gotten it to work with the debugger, then LGTM! 👻

File: Loop.xcodeproj/xcshareddata/xcschemes/Complication - WatchApp.xcscheme (hunk 1 above)
Comment (rickpasetto) [reply Loop.xcodeproj/xcshareddata/xcschemes/Complication - WatchApp.xcscheme]: In the iOS simulator or IRL?

Comment (darinkrauss) [reply Loop.xcodeproj/xcshareddata/xcschemes/Complication - WatchApp.xcscheme]: In the simulator.

Comment (rickpasetto) [reply Loop.xcodeproj/xcshareddata/xcschemes/Complication - WatchApp.xcscheme]: It seems to work the same as 11.7: namely, I can launch Loop in the iPhone simulator, and it works.  When I launch the Watch app in the debugger, it starts, but it "quits" the prior iPhone simulator run of Loop (11.7 did this for me too).  But the watch and iphone are stil linked, apparently.  Is that what you mean?

Comment (rickpasetto) [reply Loop.xcodeproj/xcshareddata/xcschemes/Complication - WatchApp.xcscheme]: Hm.. oddly, after testing it in the iOS simulator(s), it rewrote these scheme files (??).  I've posted updates

**Review by darinkrauss — APPROVED** [review]
LGTM! 🏀

**Pull request merged**